specs/schemes/exact/scheme_exact_algo.md

```diff
@@ -0,0 +1,173 @@
+# Scheme: `exact` on `Algorand`
+
+## Summary
+
+The `exact` scheme on Algorand uses the `Algorand Standard Asset (ASA)`, which are native assets (no contract required) on the Algorand blockchain to authorize a transfer of a specific amount from the payer to the resource server. The approach results in the facilitator having no ability to direct funds anywhere but the address specified by the resource server in `paymentRequirements`.
+
+## Payment Requirements asset definition 
+
+In the `exact` scheme on Algorand, the `paymentRequirements.asset` must be a string representing an `Algorand Standard Asset (ASA)` ID unsigned integer instead of an `ERC20` contract address. This **MUST** be validated by the resource server to ensure the `asset` field is valid when using the Algorand scheme.
```

Comment:
What about ALGO? That is not an ASA ID.

Reply:
x402 is for stablecoins only afaik, it excludes "gas" tokens. If needed we can make algo == 0?